.gitignore

```diff
@@ -12,3 +12,4 @@ nitter
 nitter.conf
 guest_accounts.json*
 dump.rdb
+./
```

Comment:
What does this do?

Reply:
oh weird I didn't see that. I removed a guest_accounts.json I had added, copilot might have done that. removed it now